app/scripts/utils/status.js

```diff
@@ -6,22 +6,9 @@ const status = {
 };
 export default status;
 
-export const queryStatus = [
-  'running',
-  'completed',
-  'failed'
-];
-
```

Comment:
Why is this getting deleted? Unused?

Reply:
I couldn't find any use of those variables in our code, so I just deleted because they were confusing me.  If you can think of a reason to keep them, please let me know.

Reply:
Nope, that's fine with me